### PR TITLE
[bitnami/external-dns] Allow specifying automountServiceAccountToken field on service account

### PR DESCRIPTION
**Pull request by joshuastern**
**Description of the change**

Allow for specifying the `automountServiceAccountToken` field on the external-dns service account.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/external-dns/README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ The following table lists the configurable parameters of the external-dns chart
...
 | `serviceAccount.name`                  | ServiceAccount to use. A name is generated using the external-dns.fullname template if it is not set                                                                                                            | `nil`                                                   |
 | `serviceAccount.annotations`           | Additional Service Account annotations                                                                                                                                                                          | `{}`                                                    |
+| `serviceAccount.automountServiceAccountToken` | Automount API credentials for a service account.                                                                                                                                         | `true`                                                  |
```
Comment (Mauraza) [inline bitnami/external-dns/README.md:208]: Why the default value is true? could be false?

**Review by Mauraza — COMMENTED** [review]
Hi @joshuastern,
Thanks a lot for the contribution! Could you take a look at the comment?

**Comment by Mauraza** [comment]
Hi @joshuastern,

Thanks again for the contribution! LGTM!

**Comment by BenCoughlan15** [comment]
Hey, looking at this, don't we now need a secret to go with this? 

The serviceAccount secret is no longer created by default.

**Comment by Mauraza** [comment]
Hi @BenCoughlan15,

We are open to a contribution... Could you be so kind as to create a PR adding the secret?
Many thanks for considering my request.

**Pull request merged**